cloudflare/import_cloudflare_load_balancer_pool_test.go

```diff
@@ -0,0 +1,37 @@
+package cloudflare
+
+import (
+	"testing"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccCloudFlareLoadBalancerPool_Import(t *testing.T) {
```

Comment:
I ran this test locally but it failed: 

```console
$ make testacc TESTARGS=" run=TestAccCloudFlareLoadBalancerPool_Import" 
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test $(go list ./... |grep -v 'vendor') -v -run=TestAccCloudFlareLoadBalancerPool_Import -timeout 120m
?       github.com/terraform-providers/terraform-provider-cloudflare    [no test files]
=== RUN   TestAccCloudFlareLoadBalancerPool_Import
=== PAUSE TestAccCloudFlareLoadBalancerPool_Import
=== CONT  TestAccCloudFlareLoadBalancerPool_Import
--- FAIL: TestAccCloudFlareLoadBalancerPool_Import (0.76s)
        testing.go:434: Step 0 error: Error applying: 1 error(s) occurred:

                * cloudflare_load_balancer_pool.c2p8b4gigl: 1 error(s) occurred:

                * cloudflare_load_balancer_pool.c2p8b4gigl: error creating load balancer pool: error from makeRequest: HTTP status 400: content "{\n  \"result\": null,\n  \"success\": false,\n  \"errors\": [\n    {\n      \"code\": 1002,\n      \"message\": \"the origin list length must be in range [1, 0]: validation failed\"\n    }\n  ],\n  \"messages\": []\n}\n"
FAIL
FAIL    github.com/terraform-providers/terraform-provider-cloudflare/cloudflare 0.787s
make: *** [testacc] Error 1
```

Reply:
@catsby I can't reproduce this in my account. I am going to do some more testing relating to plan levels - but I thought you were also using an enterprise account?

Reply:
@catsby having done some reading up it seems this error message is the one that occurs when you didn't enable load balancing in the account (https://community.cloudflare.com/t/cloudflare-warp-beta/5656/252). You will need to enable load balancing in the cloudflare dashboard